packages/dotlottie-js/src/tests/utils-node.spec.ts

```diff
@@ -182,6 +185,20 @@ describe('getImage', () => {
   });
 });
 
+describe('getAudio', () => {
+  it('returns the audio', async () => {
+    const audio = await getAudio(dotLottieAnimationWithAudio, 'audio_1.mpeg');
+
+    expect(audio?.length).toBeGreaterThan(0);
+  });
```

Comment:
Asserting on the length, doesn't guarantee that it return a valid audio, is it possible to check if the actual `audio_1.mpeg` equal to the extracted `audio` ?